.github/workflows/container.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: google-github-actions/setup-gcloud@v0.2.0
+      - uses: google-github-actions/setup-gcloud@v2
```

Comment:
I believe you need to auth first before you can setup gcloud with V2:
https://github.com/google-github-actions/setup-gcloud?tab=readme-ov-file#usage

We had this error on Friday where without authenticating, setup-gcloud fails (though GH action allowed you to continue unauthenticated).

Reply:
Reverted back to v0.2.0